src/components/actions/getApplicationData.ts

```diff
@@ -23,28 +23,26 @@ export const getApplicationData = async (input: {
       trigger_payload?.record_id as number
     ))
 
-  const applicationResult = await DBConnect.getApplicationData(applicationId)
-  if (!applicationResult) throw new Error("Can't get application data")
-
-  const applicationData = applicationResult ? applicationResult : { applicationId }
+  const applicationData: BaseApplicationData = await DBConnect.getApplicationData(applicationId)
+  if (!applicationData) throw new Error("Can't get application data")
 
   const {
-    user: {
-      firstName,
-      lastName,
-      organisation: orgName,
-      username,
-      dateOfBirth,
-      email,
-      permissionNames,
-    },
+    user: { firstName, lastName, organisation, username, dateOfBirth, email, permissionNames },
   } = await getUserInfo({
     userId: applicationData.userId,
-    username: applicationData.username,
```

Comment:
We should also send username, since `getUserInfo` will send to `getUserOrgData` and that is getting by either one: orgId or Username

Reply:
You only need to send one of `userId` or `username`, not both. `getUserOrgData` handles either :)